api/v1alpha1/dspipeline_types.go

```diff
@@ -40,8 +40,7 @@ type DSPASpec struct {
 	// ObjectStorage specifies Object Store configurations, used for DS Pipelines artifact passing and storage. Specify either the your own External Storage (e.g. AWS S3), or use the default Minio deployment (unsupported, primarily for development, and testing) .
 	// +kubebuilder:validation:Required
 	*ObjectStorage `json:"objectStorage"`
-	// +kubebuilder:validation:Optional
-	// +kubebuilder:default:={deploy: false}
+	// +kubebuilder:default:={deploy: true}
```

Comment:
can you change: 
```
	*MLMD `json:"mlmd"`
```
to: 
```
	*MLMD `json:"mlmd,omitempty"`
```

Reply:
this will allow us to have a dspa that's like: 

```
apiVersion: datasciencepipelinesapplications.opendatahub.io/v1alpha1
kind: DataSciencePipelinesApplication
metadata:
  name: sample
spec:
  objectStorage:
    minio:
      image: ..
  mlpipelineUI:
    image: ..
```

i.e. leave out the `mlmd: {} ` requirement